src/test/unit/utils/fact-api.ts

```diff
@@ -0,0 +1,22 @@
+import { FactApi } from '../../../main/utils/fact-api';
+import axios from 'axios';
+
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe('FactApi', () => {
+  test('Should return results from get request', async () => {
+    const results: { data: {}[]} = {
+      data: [{
+        name: 'London',
+        slug: 'London',
+        address: 'Address Street',
+        'town_name': 'AAA',
```

Comment:
I think this would change as we will go to camelCase

Reply:
fair, i just took the json file from the prototype and served that